RELEASE_NOTES

```diff
@@ -1,3 +1,4 @@
+- Refactor: Migration of bibliographic references from value types to domain entities [#qubIT-Fenix-1595] 
```

Comment:
I think there is a rebase needed.. and in that case the release note can be the same as the previous